gemviz/resources/mainwindow.ui

```diff
@@ -31,15 +31,21 @@
        </spacer>
       </item>
       <item>
-       <widget class="QLabel" name="server_uri">
+       <widget class="QLabel" name="server_label">
         <property name="toolTip">
          <string>URL of the selected tiled server.</string>
         </property>
         <property name="text">
-         <string>tiled server URI</string>
+         <string>Tiled server:   </string>
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QComboBox" name="server_uri"/>
+       <property name="toolTip">
```

Comment:
When opening this `.ui` file in the Qt designer:
![image](https://github.com/BCDA-APS/gemviz/assets/2279984/b729d4c8-70c1-4789-a123-1d5a1aff78dd)

Reply:
Was this added by editing the `.ui` file with a text editor?  When tooltip is added with designer, this is the code:

```xml
       <widget class="QComboBox" name="server_uri">
        <property name="toolTip">
         <string>Select a tiled server.</string>
        </property>
       </widget>
```

Reply:
ooops yes I updated the toolbox from the `.ui` file. What is that giving an error?

Reply:
done